config.json.example

```diff
@@ -2,6 +2,7 @@
     "max_open_trades": 3,
     "stake_currency": "BTC",
     "stake_amount": 0.05,
+    "fiat_currency": "USD",
```

Comment:
I'm not sure about the naming, because it is very similar to `stake_currency` and could lead to confusion, but right now I don't have a better naming suggestion.

Reply:
`fiat_display_currency` ?

Reply:
I like `fiat_display_currency`, I will change for it

Reply:
Done